compiler/src/dotty/tools/dotc/transform/SymUtils.scala

```diff
@@ -145,39 +163,39 @@ object SymUtils:
     *     and also the location of the generated mirror.
     *   - all of its children are generic products, singletons, or generic sums themselves.
     */
-    def whyNotGenericSum(declScope: Symbol)(using Context): String =
+    def whyNotGenericSum(using Context): String =
```

Comment:
I am not sure why `declScope` is no longer needed. I assume it's OK. But maybe add an explanation to he commit or PR?

Reply:
added an explanation to the PR